docs/topics/README.md

```diff
@@ -37,21 +36,25 @@ int main() {
 }
 ```
 
-To connect a session you need to create a [`CassCluster`](http://datastax.github.io/cpp-driver/api/struct_cass_cluster/) object and configure it. The minimal configuration needed to connect is a list of contact points. The contact points are used to initialize the driver and it will automatically discover the rest of the nodes in your cluster. It's a good idea to include more than one contact point to be robust against node failures.
+To connect a session, a [`CassCluster`](http://datastax.github.io/cpp-driver/api/struct_cass_cluster/) object will need to be created and configured. The minimal configuration needed to connect a session is a single or multiple list of contact points. The contact points are used to initialize the driver and it will automatically discover the rest of the nodes in your cluster.
```

Comment:
Yeah ... not a big fan of the wording either.  I think the original wording was good enough.  Will revert that wording.